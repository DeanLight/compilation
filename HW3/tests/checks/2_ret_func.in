byte fii(byte x, byte y)
{
	return x-y;
	return x*y;
}

int int foo(int x,int y)
{
	return x+y;
}

void main ()
{
	int x;
	int y;
	Byte z;
	z = foo(x,y);
}