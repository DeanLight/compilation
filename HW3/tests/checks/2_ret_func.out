---end scope---
x BYTE -1
y BYTE -2
line 7: syntax error
